diff --git a/credsweeper/ml_model/ml_validator.py b/credsweeper/ml_model/ml_validator.py
index 589a4bb6..a3cab6fd 100644
--- a/credsweeper/ml_model/ml_validator.py
+++ b/credsweeper/ml_model/ml_validator.py
@@ -3,7 +3,9 @@ import logging
 import string
 from pathlib import Path
 from typing import List, Tuple, Union, Optional, Dict
+import warnings
 
+warnings.filterwarnings("ignore", message="Unsupported Windows version (2022server). ONNX Runtime supports Windows 10 and above, only.", module="onnxruntime.capi.onnxruntime_validation")
 import numpy as np
 import onnxruntime as ort

